tools/version-tracker/pkg/constants/constants.go

```diff
@@ -284,6 +284,10 @@ var (
 			SourceOfTruthFile:     ".github/workflows/ci.yaml",
 			GoVersionSearchString: `GO_VERSION: "(1\.\d\d)"`,
 		},
+		"tinkerbell/rufio": {
+			SourceOfTruthFile:     ".github/workflows/ci.yaml",
+			GoVersionSearchString: `GO_VERSION: "(1\.\d\d)"`,
```

Comment:
This needs to be [single quotes](https://github.com/tinkerbell/rufio/blob/main/.github/workflows/ci.yaml#L12).